virttest/migration.py

```diff
@@ -700,11 +700,12 @@ def check_result(self, result, params):
 
         if status_error:  # Migration should fail
             if err_msg:   # Special error messages are expected
-                if not re.search(err_msg, result.stderr_text.strip()):
-                    raise exceptions.TestFail("Can not find the expected "
-                                              "patterns '%s' in output '%s'"
-                                              % (err_msg,
-                                                 result.stderr_text.strip()))
+                err_msg_list = err_msg.split('|')
```

Comment:
'|' means 'OR' for a regular expression, why do we need split the string to list here? 
We already have some cases which have multiple error messages, please check them again. Thanks!
```
migration/migrate_vm.cfg:                                    err_msg = "error: internal error: unable to execute QEMU command '(nbd-server-add|block-export-add)': Block node is read-only"
migration/migrate_options_shared.cfg:                                    err_msg = "error: operation failed: migration out job:|client socket is closed"
```

Reply:
You are right.